### PR TITLE
[Network] Add New-AzExpressRoutePortLOA cmdlet

### PR DESCRIPTION
**Pull request by 3vilbuff3r**
## Description
Added new cmdlet New-AzExpressRoutePortLOA in Microsoft.Network.
Powershell [design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/575)

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by isra-fel** [comment]
Hi Neel, please let us know when this is ready for review. Thanks.

**Comment by neelbhavsar97** [comment]
Hi @isra-fel , the code in this PR depends on a swagger change in network-june-release branch. Some how if I merge network-june powershell with my code then the version 20.0.2-preview of SDK does not seem to have the swagger change while the 20.0.1-preview has the change that we did. I am currently waiting for the swagger to get merged with master. Please guide if you think there's something I might be missing.

**Comment by isra-fel** [comment]
No problem. I was just reminding you about this PR 😁

**Comment by isra-fel** [comment]
Any updates? @neelbhavsar97

**Comment by neelbhavsar97** [comment]
@isra-fel , sorry for the delayed reply. I was waiting for Az Network team to complete the backend roll out. I have re-targeted the PR to master. My swagger changes are merged with master branch of azure-rest-api-specs but for some reason the nuget used by master branch of powershell does not seem to hove those changes. Which SDK is generally used by powershell? and why would the swagger master changes be not there?

**Comment by isra-fel** [comment]
Hi @neelbhavsar97 I believe you need to wait until #12751 is merged then sync with master.

**Comment by neelbhavsar97** [comment]
@isra-fel , Is there an ETA on when that will be merged?

**Comment by neelbhavsar97** [comment]
@isra-fel , all checks are passing and network-june is merged with master. Please go ahead and merge this to master as well. Let me know if you need anything from our side.

File: src/Network/Network/ChangeLog.md
Comment (msJinLei) [inline src/Network/Network/ChangeLog.md:60]: Please fix the changelog

File: .github/CODEOWNERS
Comment (msJinLei) [inline .github/CODEOWNERS:3]: Please fix CODEOWNERS

**Review by msJinLei — CHANGES_REQUESTED** [review]

**Comment by neelbhavsar97** [comment]
@msJinLei , thank you for identifying the merge markers that I had missed. Let me know if there is any other change required. :)

**Review by isra-fel — APPROVED** [review]

**Pull request merged**
